made website responsive

ArkanKovanAbdullah opened the pull request:
## Made website responsive

- Added media queries.
- Made some of the dimensions in percentages.
- Made the pictures change when in desktop version.

vickymarz commented on index.html:
```diff
@@ -125,60 +127,69 @@ <h1 id="my-name">I’m Arkan, Glad to see you!</h1>
-        </div>
-        <!-- Contact section -->
+        </section>
         <section class="contact-me-bg" id="contact-me">
```
- [ ] Kindly adjust the width of your contact form to match the Figma design you chose so that your UI can look better 😄 . 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/193349130-484eba8c-32ef-4f7f-b125-6a37b2595260.png)  | ![image](https://user-images.githubusercontent.com/63473584/193349476-4bb07eda-c10f-45e8-8d20-e978cbbc472d.png)  |

vickymarz commented on index.html:
```diff
@@ -125,60 +127,69 @@ <h1 id="my-name">I’m Arkan, Glad to see you!</h1>
-                <div id="title-box">
-                    <h1 id="my-name">About <br>Me</h1>
+
+        <section class="about-myself" id="about-myself">
```
- [ ] Kindly ensure that all the texts in your about-me section match the Figma design you chose in terms of color, font size, font family, and font weight so that your UI can look better 😄 . YOu can check the Figma design CSS properties for more insights 👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/193350215-6892fe6e-7533-4bc9-8ac2-6cb39511416c.png)  | ![image](https://user-images.githubusercontent.com/63473584/193350446-4cd5e516-eeb9-4c7e-a815-9ce65e0d12ec.png)  |

Alfredbis29 replied:
- [ ] Kindly change this as required in the previous review :wink:

vickymarz commented on index.html:
```diff
@@ -64,7 +66,7 @@ <h1 id="my-name">I’m Arkan, Glad to see you!</h1>
             </div>
             <div id="section-two-one">
-                <div class="projects-card-two"></div>
+                    <div class="projects-card-two"></div>
```
- [ ] I think you should consider reducing the spacing between all your project card description so that your UI can look better  👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/193351337-53f70868-b442-47ec-bac3-ad52bbad2397.png)  |![image](https://user-images.githubusercontent.com/63473584/193351412-77e5ef74-51c4-4a79-a68f-26f1f65e5cfc.png)  |

vickymarz commented on index.html:
```diff
@@ -14,32 +14,34 @@
-                <div id="title-box">
-                    <h1 id="my-name">I’m Arkan, Glad to see you!</h1>
+        <section id="section-one">
+            <div class="desktop-me">
```
- [ ] The background-attachment of your background is set to fixed which is not making your UI look very good. Kindly remove it from fixed  👍🏽 

![image](https://user-images.githubusercontent.com/63473584/193351982-7229a837-ffa3-40fe-a66d-07c18abed380.png)

ArkanKovanAbdullah replied:
@vickymarz I did it cause I though it look nice. But on second thoughts you are actually right it doesn't. 


I removed now. Thanks....

Alfredbis29 replied:
- [ OPTIONAL] You made great effort to make some changes from the previous review but you still need working on something but you are almost there , kindly improve your design as requested by the previous review

This is how your Design looks after changement :point_down: 
> Your Design
> 
![desktop122](https://user-images.githubusercontent.com/88894525/193361784-83b20118-ea97-43a7-a37a-53b2ee4e747a.png)

Figma Design	
![figmaDesktop](https://user-images.githubusercontent.com/88894525/193361863-e247fe9d-5e22-4533-ac79-e4dd82d8b1b8.png)

vickymarz reviewed (changes requested):
Hi @ArkanKovanAbdullah, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._
<img src="https://media2.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Alfredbis29 reviewed (changes requested):
> # _Status: Required Changes_ 
> Hi @Habtamu210 ,
> 
> Good job so far for making some changes! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
 * No linter errors. :heavy_check_mark:
 * Good README.  :heavy_check_mark:
 * Correct GitHub Flow.  :heavy_check_mark:
 * Good commit messages.  :heavy_check_mark:
 
![](https://user-images.githubusercontent.com/20554583/192241479-b6304cb0-f256-4c64-a258-4f5a40a1e960.gif)
 
 ## Required Changes 
 

- [ ] _Check the comments under the review._

 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
 
I wish you the best in making these changes and in your subsequent projects. heart

ArkanKovanAbdullah commented:
@Alfredbis29 Please recheck, I have done these changes, But I believe your browser still doesn't have Poppins font installed, that's why you cant see them perfectly look alike. 

# My css code:
![01 10 2022_01 37 34_REC](https://user-images.githubusercontent.com/79310452/193365663-e9c35bd3-3902-4d3b-b721-d01561ba1133.png)


# Figma Template:
![01 10 2022_01 37 26_REC](https://user-images.githubusercontent.com/79310452/193365686-486e7196-3ae0-441e-80ae-8e0e5cb0932f.png)

Alfredbis29 commented:
- [ ]Hello @ArkanKovanAbdullah, I get you , but this is what we can see :point_down: 
![aboutpage](https://user-images.githubusercontent.com/88894525/193366054-e28ec78d-a884-46fe-908f-3807800ea373.png)

ArkanKovanAbdullah commented:
@Alfredbis29 The color of about Myslef was not like Figma but I updated that and pushed again hoping that you would approve it. 
Also the text in my web said 'about me' but in Figma says 'About Myself' But that also i fixed and pushed with the hope the hope that you would approve it. 


# Before first review:
![193350446-4cd5e516-eeb9-4c7e-a815-9ce65e0d12ec](https://user-images.githubusercontent.com/79310452/193366075-01b2213f-b3be-4e8b-8ac3-2469ae2039e7.png)

# After first review:
![01 10 2022_01 45 09_REC](https://user-images.githubusercontent.com/79310452/193366162-604dd9e4-f604-4342-a103-dd22cb0537eb.png)

ArkanKovanAbdullah commented:
> > # _Status: Required Changes_
> > Hi @Habtamu210 ,
> > Good job so far for making some changes! There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ## Highlights
> * No linter errors. ✔️
> * Good README.  ✔️
> * Correct GitHub Flow.  ✔️
> * Good commit messages.  ✔️
> 
> ![](https://user-images.githubusercontent.com/20554583/192241479-b6304cb0-f256-4c64-a258-4f5a40a1e960.gif) [ ![192241479-b6304cb0-f256-4c64-a258-4f5a40a1e960.gif](https://user-images.githubusercontent.com/20554583/192241479-b6304cb0-f256-4c64-a258-4f5a40a1e960.gif) ](https://user-images.githubusercontent.com/20554583/192241479-b6304cb0-f256-4c64-a258-4f5a40a1e960.gif) [ ](https://user-images.githubusercontent.com/20554583/192241479-b6304cb0-f256-4c64-a258-4f5a40a1e960.gif)
> 
> ## Required Changes
> * [ ]  I encourage you to follow the requirement given by the previous review 😉
> * [ ]  _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding! 👏 👏 👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
> 
> I wish you the best in making these changes and in your subsequent projects. heart


> * [ ]  Hello @ArkanKovanAbdullah, I get you , but this is what we can see 👇
> 
> ![aboutpage](https://user-images.githubusercontent.com/88894525/193366054-e28ec78d-a884-46fe-908f-3807800ea373.png)

@Alfredbis29 Yes maybe you don't have Poppins font installed.

Alfredbis29 commented:
- [ ] Hello @ArkanKovanAbdullah , If you push after the review you can ask for another opinion , but on my side I have everything installed , you can ask for the second opinion please . :wink:  Good luck :wink:

ArkanKovanAbdullah commented:
> * [ ]  Hello @ArkanKovanAbdullah , If you push after the review you can ask for another opinion , but on my side I have everything installed , you can ask for the second opinion please . 😉  Good luck 😉

@Alfredbis29 Please check if you have installed Poppins font. And please this video which shows why there is no problem regarding the font. 
many thanks...

https://user-images.githubusercontent.com/79310452/193367361-89fdf6db-7b6b-4821-b242-18b047c317ea.mp4

Olamarx reviewed (approved):
Hi @ArkanKovanAbdullah, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good commit messages ✔️ 
- Linters passed 👍🏼 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

rukundoeric commented:
Hello @aizjicod  👋,

_This is an additional review as a result of the second opinion request._

I see the issue is resolved already. 

Keep rocking 🚀!
